### ISSUE TITLE
Extension doesn't work on remote workspaces

### ISSUE THREAD

**Issue by pippinsoft** [ManuelHentschel/vscode-github-linker#7]
The extension works great on local workspaces (thanks!!!).

On remote workspaces I get the following error:

`Could not copy GitHub URL: Git extension not installed or not activated yet!`

I confirmed that the builtin git extension is installed globally. 

This is true for multiple workspaces, both local (works) and remote (error), across multiple local and remote machines (all running Linux). Please let me know if I can provide any further information.

**Comment by ManuelHentschel** [comment]
I wasn't able to reproduce on WSL, but I found the option to add an "extensionDependency" in the description of the [vscode.git extension](https://github.com/microsoft/vscode/blob/4a3ed95527eea750a62ad484d61c08baae571285/extensions/git/README.md) and fixed a deprecation warning. These changes are published (v0.0.12), does this update change anything w.r.t. your issue?

**Comment by pippinsoft** [comment]
Thanks for the quick response, but that did not work. I noticed that VSCode reports that the extension is running locally because it prefers to do so. I removed `ui` from `extensionKind` and that solved the problem.

https://github.com/ManuelHentschel/vscode-github-linker/blob/cd2e189f3e03a28e48bf999fcfd1d3ffcad7f727/package.json#L9-L12

Perhaps switching to `"extensionKind": ["workspace", "ui"]` as described here: https://code.visualstudio.com/api/advanced-topics/extension-host#preferred-extension-location would work; I have not yet tested that.
